portfolio.css

```diff
@@ -0,0 +1,578 @@
+}
+
+.port__description {
+	font-size: 20px;
+	margin-top: 5px;
+	text-align: left;
+}
+
+.portfolio__image {
+	padding: 10px;
+	margin-top: 20px;
+	margin-bottom: 5px;
+	border: solid white 5px;
+	width: 100%;
+	object-fit: scale-down;
+}
+
+.contact {
+	color: #ffffff;
+	background-color: #EF1B1B;
+	opacity: 80%;
+	padding: 50px 30px 30px 30px;
+	margin: 20px 20px 20px 20px;
+    object-fit: cover;
+    border-radius: 1%;
+	text-align: center;
+}
+
+.contact h1 {
+	margin-top: 20px;
+	margin-bottom: 40px;
+	padding: 20px;
+	border: white solid 1px;
+	font-size: 70px;
+	display: inline;
+}
+
+form {
+	text-align: left;
+	margin: 60px 150px 20px 150px;
+}
+
+label {
+	font-size: 20px;
+	margin-bottom: 10px;
+}
+
+input[type="text"] {
+	display: block;
+	border: white solid 1px;
+	border-radius: 5px;
+	width: 100%;
+	margin-top: 5px;
+	margin-bottom: 20px;
+}
+
+input[type="submit"] {
+	font-size: 20px;
+	width: 150px;
+	padding: 10px;
+	font-family: 'Major Mono Display';
+	background-color: white;
+	border: none;
+	border-radius: 5px;
+	margin: 0 auto;
+	display: block;
+}
+
+form .message {
+	height: 100px;
+	margin-bottom: 40px;
+}
+
+@media screen and (max-width: 1300px) {	
+	.intro {	
+		margin: 20px 250px 20px 20px;
+	}	
+		
+	.profile__picture {
+		object-fit: scale-down;
+	}	
+}
+
+@media screen and (max-width: 1270px) {
+	.gray__box {
+		color: #C8C8C8;
+		background-color: #C8C8C8;
+		height: 212px;
+		width: 30%;
+		position: absolute;
+		top: 0%;
+		right: 0%;
+	}
+	
+	.links {
+		margin-right: 500px;
+	}
+	
+	.intro__text p {
+			font-size: 37px;
+		}
+	.about {	
+		margin: 20px 150px 20px 20px;
+	}
+	
+	form {
+		margin: 60px 100px 20px 100px;
+	}
+		
+}
+
+@media screen and (max-width: 1185px) {
+	.intro {	
+		margin: 20px 200px 20px 20px;
+	}	
+}		
+
+
+@media screen and (max-width: 1125px) {
+		
+	.gray__box {
+		width: 20%;
+	}
+	
+	.intro {	
+			margin: 20px 150px 20px 20px;
+		}
+		
+	.intro__text p {
+			font-size: 35px;
+		}
+	
+	.about {	
+		margin: 20px 100px 20px 20px;
+	}
+	
+	.portfolio {
+		margin: 20px 50px 20px 20px;
+	}
+	
+	.portfolio__content {
+		grid-template-columns: 1fr 1fr;
+	}
+	
+	.port__description {
+		font-size: 20px;
+	}
+	
+	form {
+		margin: 60px 50px 20px 50px;
+	}
+	
+	.about li {
+		font-size: 35px;
+	}
+}
+
+@media screen and (max-width: 1035px) {
+	.intro__text p {
+		font-size: 30px;
+	}
+}
+
+
+@media screen and (max-width: 970px) {
+	.gray__box {
+		width: 10%;
+	}	
+	
+	.intro {
+		margin: 20px 85px 20px 20px;
+	}
+	
+	.intro h1 {
+		margin: 10px;
+		font-size: 60px;
+	}
+	
+	.about {	
+		margin: 20px 70px 20px 20px;
+	}
+	
+	.portfolio {
+		margin: 20px 50px 20px 20px;
+	}
+	
+	.port__description {
+		font-size: 15px;
+	}
+	
+	.about li {
+		font-size: 30px;
+	}
+}
+
+@media screen and (max-width: 880px) {
+	.gray__box {
+			opacity: 0%;
+	}	
+	
+	.intro h1 {
+			font-size: 60px;
+	}
+	
+	.intro {
+		margin: 20px;
+	}
+	
+	.about {	
+		margin: 20px;
+	}
+	
+	.portfolio {
+		margin: 20px;
+	}
```

Comment:
fair bit of repetition here, could a more general rule be made earlier in the sheet?